bubblesub/model.py

```diff
@@ -229,6 +236,13 @@ def __iter__(self) -> T.Iterator[TItem]:
         """
         yield from self._items
 
+    def __reversed__(self) -> T.Iterator[TItem]:
+        """Iterate reversly over the collection values.
```

Comment:
Thanks. `Iterate over … in a reverse order.` and I'd move it under `__iter__`

Reply:
It's already under `__iter__`

Reply:
Sorry it appeared like it was under `__get__`